src/types/request.ts

```diff
@@ -31,6 +31,7 @@ export interface IRequestComponent {
     actions: {
         setDimensions: Function;
     };
+    permissionModeType: boolean;
```

Comment:
Please see one of my comments above. Then we could remove `permissionModeType` here?

Reply:
75764e625bcc3e45ab338fb1ad0d5b0e24f5fd94